.chloggen/exporter-helper-move-request.yaml

```diff
@@ -0,0 +1,18 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: exporterhelper
```

Comment:
You can avoid this for the moment, by duplicating the interfaces for the moment since that works in go. I think is better to postpone a bit this, since I think with the new Queue interface we can make the current internal depend on the main package not the other way around.

Reply:
If you don't like the duplicate code, using aliases also is an option.